src/pages/home/report/ReportActionCompose.js

```diff
@@ -381,10 +379,18 @@ class ReportActionCompose extends React.Component {
      */
     updateComment(comment, shouldDebounceSaveComment) {
         const newComment = EmojiUtils.replaceEmojis(comment);
-        this.setState({
-            isCommentEmpty: !!newComment.match(/^(\s|`)*$/),
-            value: newComment,
-        });
+        const isCommentEmpty = Boolean(newComment.match(CONST.REGEX.IS_COMMENT_EMPTY));
+        if (this.state.isCommentEmpty !== isCommentEmpty) {
+            this.setState({isCommentEmpty});
+        }
+
+        // When the comment has changed after replacing emojis we need to update the text in the input
+        if (newComment !== comment) {
+            const lengthDiff = newComment.length - comment.length;
+
+            // we assume that at the last position of our text a emoji has been added, thus we have to add a offset of 1
+            this.textInput.setTextAndSelection(newComment, this.selection.start + lengthDiff + 1, this.selection.end + lengthDiff + 1);
+        }
```

Comment:
This logic is only about adding/replacing emojis using text like `:smile:` correct?

Assuming emoji was added at the end is a bit concerning - what would happen if someone writes an emoji like `:cat:` in the middle of a draft? 

I think `EmojiUtils.replaceEmojis` should handle such side effects as it is the one that replacing words with emojis